packages/composite-checkout/src/wpcom/wp-checkout-order-summary.js

```diff
@@ -23,8 +23,8 @@ export default function WPCheckoutOrderSummary() {
 		<React.Fragment>
 			<SummaryContent>
 				<ProductList>
-					{ items.map( product => {
-						return <ProductListItem key={ product.id }>{ product.label }</ProductListItem>;
+					{ items.map( ( product, index ) => {
+						return <ProductListItem key={ product.label }>{ product.label }</ProductListItem>;
```

Comment:
Are you sure that two products won't have the same label? The [schema for line items](https://github.com/Automattic/wp-calypso/blob/master/packages/composite-checkout/README.md#checkoutprovider) requires that each item have a unique `id` property, so using that might be best.